tests/test_inclined_exponential.py

```diff
@@ -16,10 +16,15 @@
 #    and/or other materials provided with the distribution.
 #
 
+"""This file contains tests for the InclinedExponential and InclinedSersic classes.
+"""
+
```

Comment:
Also, I think you should add tests for both inclined profiles to ```test_config_gsobject.py``` to make sure that the config initialization works properly.